generators/client/templates/angular/src/main/webapp/_index.html

```diff
@@ -1,6 +1,15 @@
 <!doctype html>
 <html class="no-js">
 <head>
+    <script>
```

Comment:
can we move this to right after the `<jhi-main></jhi-main>` in this file please.

Btw did this work in IE?
@holgerdunst does this solve your issue?